istio/install.go

```diff
@@ -37,10 +49,36 @@ func (istio *Istio) installIstio(del bool, version, namespace string) (string, e
 		return st, ErrMeshConfig(err)
 	}
 
-	err = istio.runIstioCtlCmd(version, del)
+	// Fetch and/or return the path to downloaded and extracted release bundle
+	dirName, err := istio.getIstioRelease(version)
+	if err != nil {
+		// ErrGettingIstioRelease
+		return st, err
+	}
+
+	// Install using istioctl if explicitly stated
+	if useBin {
+		err = istio.runIstioCtlCmd(version, del, dirName)
+		if err != nil {
+			//ErrInstallUsingIstioCtl
```

Comment:
Shall we fallback to the other if either fails?

Reply:
@leecalcote do we have to support both istioctl and chart based installations?